clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/SerializeApplicationAtomicOperation/RestoreSnapshotAtomicOperation.groovy

```diff
@@ -151,25 +229,50 @@ class RestoreSnapshotAtomicOperation implements AtomicOperation<Void> {
       sleep(1000)
       jobStatus = jobExecutor.updateJob(jobId)
     }
-
-    if (jobStatus.result == JobStatus.Result.FAILURE && jobStatus.logsContent) {
+    if (jobStatus.result == JobStatus.Result.FAILURE && jobStatus.stdOut) {
       cleanUpDirectory()
-      throw new IllegalArgumentException(jobStatus.logsContent)
+      throw new IllegalArgumentException("$jobStatus.stdOut + $jobStatus.stdErr")
+    }
+  }
+
+  private void createTerraformConfig() {
+    Map snapshot = front50Service.getSnapshotVersion(directory, snapshotTimestamp.toString())
+    Map terraformConfig = [:]
+    terraformConfig.resource = snapshot.infrastructure
+    if (!hasEnvCredentials) {
+      Map provider = [:]
+      provider.google = [:]
+      provider.google.credentials = "\${file(\"$credentialPath\")}".toString()
+      provider.google.project = project
+      provider.google.region = "us-central1"
```

Comment:
Please add a comment explaining why this default value is required, and what the implications are.